systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlST.java

```diff
@@ -155,6 +196,23 @@ void testCruiseControlWithApiSecurityDisabled() {
     @IsolatedTest("Using more tha one Kafka cluster in one namespace")
     @Tag(SANITY)
     @Tag(ACCEPTANCE)
+    @TestDoc(
+        description = @Desc("Using multiple Kafka clusters within a single namespace to test Cruise Control with rebalance resource and refresh annotation."),
+        steps = {
+            @Step(value = "Create test storage from the test context", expected = "Test storage is initialized"),
```

Comment:
```suggestion

```
I would remove the test storage step everywhere, as it's not relevant or important for the test case.